lib/compat/wordpress-6.1/block-template-utils.php

```diff
@@ -305,6 +414,84 @@ function gutenberg_build_block_template_result_from_post( $post ) {
 			$template->area = $type_terms[0]->name;
 		}
 	}
+	if ( 'wp_template' === $post->post_type && empty( $template->description ) && ( empty( $template->title ) || $template->title === $template->slug ) ) {
+		$matches = array();
+		if ( preg_match( '/(author|page|single|tag|category|taxonomy)-(.+)/', $template->slug, $matches ) ) {
+			$type           = $matches[1];
+			$slug_remaining = $matches[2];
+			switch ( $type ) {
+				case 'author':
+					$nice_name = $slug_remaining;
+					$users     = get_users(
```

Comment:
I haven't checked the query well here, but in my testing when I updated a nicename of a contributor user, it didn't find the user.. Can you reproduce?

Reply:
Issue fixed 👍